SmartDeviceLinkTests/RPCSpecs/RequestSpecs/SDLSubscribeVehicleDataSpec.m

```diff
@@ -146,160 +150,172 @@
+context(@"initWithAccelerationPedalPosition:airbagStatus:beltStatus:bodyInformation:cloudAppVehicleID:clusterModeStatus:deviceStatus:driverBraking:eCallInfo:emergencyEvent:engineOilLife:engineTorque:externalTemperature:fuelLevel:fuelLevelState:gps:headLampStatus:instantFuelConsumption:myKey:odometer:prndl:rpm:speed:steeringWheelAngle:tirePressure:wiperStatus:", ^{
+    SDLSubscribeVehicleData* testRequest = [[SDLSubscribeVehicleData alloc] initWithAccelerationPedalPosition:YES airbagStatus:YES beltStatus:YES bodyInformation:YES cloudAppVehicleID:YES clusterModeStatus:YES deviceStatus:YES driverBraking:YES eCallInfo:YES electronicParkBrakeStatus:YES emergencyEvent:YES engineOilLife:YES engineTorque:YES externalTemperature:YES fuelLevel:YES fuelLevelState:YES fuelRange:YES gps:YES headLampStatus:YES instantFuelConsumption:YES myKey:YES odometer:YES prndl:YES rpm:YES speed:YES steeringWheelAngle:YES tirePressure:YES turnSignal:YES wiperStatus:YES handsOffSteering:YES];
 
-        expect(testRequest.accPedalPosition).to(equal(@YES));
-        expect(testRequest.airbagStatus).to(equal(@YES));
-        expect(testRequest.beltStatus).to(equal(@YES));
-        expect(testRequest.bodyInformation).to(equal(@YES));
-        expect(testRequest.cloudAppVehicleID).to(equal(@YES));
-        expect(testRequest.clusterModeStatus).to(equal(@YES));
-        expect(testRequest.deviceStatus).to(equal(@YES));
-        expect(testRequest.driverBraking).to(equal(@YES));
-        expect(testRequest.eCallInfo).to(equal(@YES));
-        expect(testRequest.electronicParkBrakeStatus).to(equal(@YES));
-        expect(testRequest.emergencyEvent).to(equal(@YES));
-        expect(testRequest.engineOilLife).to(equal(@YES));
-        expect(testRequest.engineTorque).to(equal(@YES));
-        expect(testRequest.externalTemperature).to(equal(@YES));
-        expect(testRequest.fuelLevel).to(equal(@YES));
-        expect(testRequest.fuelLevel_State).to(equal(@YES));
-        expect(testRequest.fuelRange).to(equal(@YES));
-        expect(testRequest.gps).to(equal(@YES));
-        expect(testRequest.headLampStatus).to(equal(@YES));
-        expect(testRequest.instantFuelConsumption).to(equal(@YES));
-        expect(testRequest.myKey).to(equal(@YES));
-        expect(testRequest.odometer).to(equal(@YES));
-        expect(testRequest.prndl).to(equal(@YES));
-        expect(testRequest.rpm).to(equal(@YES));
-        expect(testRequest.speed).to(equal(@YES));
-        expect(testRequest.steeringWheelAngle).to(equal(@YES));
-        expect(testRequest.tirePressure).to(equal(@YES));
-        expect(testRequest.turnSignal).to(equal(@YES));
-        expect(testRequest.wiperStatus).to(equal(@YES));
+        it(@"all set", ^{
```

Comment:
```suggestion
        it(@"should set all the parameters properly", ^{
```

Reply:
"should set all the parameters to YES" - maybe it is better to be more specific